src/game_event.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum, auto
+
+
+# TODO(DP): add positive & negative effects
+class GameEvent(Enum):
+    CONTINUE = auto()
```

Comment:
So `PAUSE` is `EngineEvent` and `CONTINUE` is `GameEvent`. And after `PAUSE` we will wait `CONTINUE`, right?

Reply:
discussed -- leave as is